CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### Fix
+
+- HTTP API: Extract API results dataframes separately and merge them after.
```

Comment:
The changelog is destined to end-users. Instead of explaining the code change, could you explain the issue that was solved ?